.github/workflows/docker.yaml

```diff
@@ -0,0 +1,38 @@
+name: Docker Image CI
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  buildx:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Docker Registry
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
```

Comment:
@dhui In order for GitHub Actions to build and push the image automatically to Docker Hub, please add `DOCKERHUB_USERNAME` and `DOCKERHUB_TOKEN` into the repository secrets.